Dockerfile

```diff
@@ -67,6 +67,7 @@ ARG RPI_KERNEL_CHECKSUM="295a22f1cd49ab51b9e7192103ee7c917624b063cc5ca2e11434164
 
 COPY --from=qemu-builder /qemu/arm-softmmu/qemu-system-arm /usr/local/bin/qemu-system-arm
 COPY --from=qemu-builder /qemu/aarch64-softmmu/qemu-system-aarch64 /usr/local/bin/qemu-system-aarch64
+COPY --from=qemu-builder /qemu/qemu-img /usr/local/bin/qemu-img
```

Comment:
We should `strip` this binary in the builder image first.

Reply:
This reduces the binary from 15.6M to  3.4M!

Reply:
Resolved via https://github.com/lukechilds/dockerpi/pull/21/commits/9809ecd0709f31de09d3cd4cbeb2cffefb29f477